src/app/centers/create-center/create-center.component.html

```diff
@@ -0,0 +1,129 @@
+          </mat-checkbox>
+
+          <mat-form-field *ngIf="this.centerForm.controls.active.value">
+            <mat-label>Activation Date</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="activationDatePicker" required formControlName="activationDate">
+            <mat-datepicker-toggle matSuffix [for]="activationDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #activationDatePicker></mat-datepicker>
+            <mat-error *ngIf="centerForm.controls.activationDate.hasError('required')">
+               Activation Date is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>External id</mat-label>
+            <input matInput formControlName="externalId">
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Submitted on</mat-label>
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" required formControlName="submittedOnDate">
+            <mat-datepicker-toggle matSuffix [for]="submittedOnDatePicker"></mat-datepicker-toggle>
+            <mat-datepicker #submittedOnDatePicker></mat-datepicker>
+            <mat-error *ngIf="centerForm.controls.submittedOnDate.hasError('required')">
+              Submission Date is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+
+          <mat-form-field>
+            <mat-label>Select and Add groups</mat-label>
+            <mat-select [formControl]="groupChoice">
+              <mat-option *ngFor="let group of groupsData" [value]="group">
+                {{ group.name }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+
+        </div>
+
+        <div fxLayout="row">
+
+          <div class="mat-table" *ngIf="groupChoice.value">
+            <div class="mat-header-row">
+              <div class="mat-header-cell">Group Details</div>
+              <button mat-icon-button (click)="addGroup()">
+                <fa-icon icon="plus"></fa-icon>
+              </button>
+            </div>
+            <div class="mat-row">
+              <div class="mat-cell">Name</div>
+              <div class="mat-cell">{{groupChoice.value.name}}</div>
+            </div>
+            <div class="mat-row">
+              <div class="mat-cell">Id</div>
+              <div class="mat-cell">{{groupChoice.value.id}}</div>
+            </div>
+            <div class="mat-row">
+              <div class="mat-cell">Office</div>
+              <div class="mat-cell">{{groupChoice.value.officeName}}</div>
+            </div>
+          </div>
+
+          <mat-nav-list *ngIf="groupMembers.length">
+            <h3 matSubheader>Selected Groups</h3>
+            <div mat-list-item *ngFor="let group of groupMembers index as i">
+              <button mat-icon-button (click)="removeGroup(i)">
+                <fa-icon icon="trash"></fa-icon>
```

Comment:
Can you place the icon on the right end of the line? That will look better.

Reply:
I think it's fine, also if I do it the row collapses dont know why, I may have to add some extra css unescessarily